app/src/main/java/ingsw/group1/findmyphone/alarm/AlarmManager.java

```diff
@@ -39,6 +39,23 @@ public String getAlarmRequestMessage() {
         return AlarmMessageHelper.composeRequestAlarm();
     }
 
+    /**
+     * Get an alarm response message composed by {@link AlarmMessageHelper}
+     *
+     * @return a formatted message for an alarm response indicating the time the device rang
+     */
+    public String getAlarmResponseMessage(double time) {
```

Comment:
I heavily suggest a long value for the time, measured in milliseconds

Reply:
Well, it's not much different, why?

Reply:
This is due to the fact It becomes easier on both ends to use a long value. You can just use `System.currentTimeMillis()` to measure it and `Calendar`+`DateUtils` to format it.

Reply:
Makes perfect sense